index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```

Comment:
Make sure you have prettier installed and your editor is configured to format html files using prettier. Check this out if you haven't: https://github.com/Hyve-Mind/fundamentals/blob/main/setting_up_vscode.md

Reply:
Prettier for vscode: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode